packages/tasit-action/src/erc721/ERC721Full.test.js

```diff
@@ -219,9 +232,7 @@ describe("TasitAction.ERC721.ERC721Full", () => {
 
       const actionErrorListener = message => {
         const { error } = message;
-        expect(error.message).to.equal(
-          "Action with error: VM Exception while processing transaction: revert"
-        );
+        expect(error.message).to.equal("Action failed.");
```

Comment:
Should we extract message from error here using destructuring assignment to be consistent?

Reply:
Done for all cases on: https://github.com/tasitlabs/TasitSDK/issues/370

Reply:
Thanks! Resolving this